ui/app/templates/components/secret-list/ssh-role-item.hbs

```diff
@@ -78,40 +76,21 @@
                   {{/if}}
                 {{/if}}
                 {{#if @item.canEditZeroAddress}}
-                  {{#if @item.zeroAddress}}
-                    <li class="action">
-                      <button
-                        type="button"
-                        disabled={{get this (concat "loading-" @item.id)}}
-                        class="link button is-transparent {{if (get this (concat 'loading-' @item.id)) 'is-loading'}} "
-                        {{on "click" @toggleZeroAddress}}
-                      >
-                        Disable Zero Address
-                      </button>
-                    </li>
-                  {{else}}
-                    <li class="action">
-                      <button
-                        type="button"
-                        disabled={{get this (concat "loading-" @item.id)}}
```

Comment:
`{{get this (concat "loading-" @item.id)}}` was undefined when i tried logging it - so passed in as an arg instead`